### PR TITLE
Fix some missing places where prioritisation is not incorporated

### PR DESCRIPTION
**Pull request by seamuslee001**
@colemanw

### CODE REVIEW

**Comment by seamuslee001** [comment]
Jenkins re test this please

**Comment by seamuslee001** [comment]
@colemanw this is ready for review now

File: CRM/ACL/BAO/ACL.php [hunk 1]
```diff
@@ -9,6 +9,8 @@
  +--------------------------------------------------------------------+
  */
 
+use Civi\Api4\Group;
```
Comment (colemanw) [inline CRM/ACL/BAO/ACL.php:12]: Unused use statement

**Comment by seamuslee001** [comment]
@colemanw have removed the unused use statement now

**Pull request merged**